daprdocs/content/en/dotnet-sdk-docs/dotnet-actors/_index.md

```diff
@@ -2,9 +2,447 @@
 type: docs
 title: "Getting started with the Dapr actors .NET SDK"
 linkTitle: "Actors"
-weight: 40000
-description: How to get up and running with the Dapr .NET SDK
+weight: 100000
+description: Try out .NET virtual actors with this example
 ---
 
 The Dapr actor package allows you to interact with Dapr virtual actors from a .NET application.
 
+## Prerequisites
```

Comment:
I think we should leave this example guide as a separate page (`dotnet-actor-howto`) instead of placing it in the `_index.md` file. These category pages should be a quick overview with an about section, that points to other content pages.

Reply:
sure, will love this back